CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
Do we need to mention this in the changelog?

Reply:
Any contributor should now how to update the changelog so I'd be keen to have it here.

Reply:
If it's for contributors, then we have the [CONTRIBUTING.md](CONTRIBUTING.md) file where you could add this.